workers-site/package.json

```diff
@@ -5,6 +5,6 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
-    "@cloudflare/kv-asset-handler": "~0.0.11"
+    "@cloudflare/kv-asset-handler": "~0.1.0"
```

Comment:
this should be `~0.1.11` to match the other one right?

Reply:
ope! yeah it'll pull 1.11 either way but yeah i can update it.